study-builder/studies/mbc/tissue-consent.conf

```diff
@@ -69,7 +69,15 @@
     user.studies["cmi-mbc"].forms["TISSUECONSENT"].questions["TISSUECONSENT_FULLNAME"].answers.hasText()
     && user.studies["cmi-mbc"].forms["TISSUECONSENT"].questions["TISSUECONSENT_DOB"].answers.hasDate()
   """,
-  "elections": [],
+  "elections": [
+    {
+      "stableId": "TISSUECONSENT",
+      "selectedExpr": """
```

Comment:
We don't have any question with stableID to track consented or not since this is specific to Tissue consent activity 
Having election this way seem to work fine.